tests/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+# The following environment variables are available to customize test execution:
+# - KUBECONFIG
```

Comment:
hm, for some reason now `make test` for me doesn't work with error 
```
docker run -i --rm \
        -v {#redacted}/kudo-spark-operator/tests:/tests \
        -v {#redacted}/kudo-spark-operator/admin.conf:/root/.kube/config \
        -e KUBECONFIG=/root/.kube/config \
        -e SPARK_IMAGE="mesosphere/spark-2.4.4-bin-hadoop2.7-k8s:2cd2dc39bde0dbcf9c2e76a94e004db01f2779fe" \
        -e SPARK_OPERATOR_IMAGE="mesosphere/kudo-spark-operator:327b0d0bbc4444f01c565c02618dd90c5f2d4a03" \
        mesosphere/spark-operator-docker-builder:b622eeb7b4f15d0297fac1400e8009e25e7c9946 \
        /bin/bash -c \
        "kubectl cluster-info && \
        echo \$SPARK_IMAGE && echo \$SPARK_OPERATOR_IMAGE"
# tests entrypoint
error: error loading config file "/root/.kube/config": read /root/.kube/config: is a directory
make: *** [Makefile:89: test] Error 1```

Looks like it's missing a mount of the kube config somewhere?

Reply:
Have you tried it with `make cluster-create`? 
If not, when using a pre-existing cluster, try setting `KUBECONFIG` explicitly when calling `make test`